TombRaider1996/EmulatorData.cs

```diff
@@ -0,0 +1,51 @@
+﻿using LiveSplit.ComponentUtil;
+using System;
+
+namespace TR1
+{
+    /// <summary>
+    /// The emulator's watched memory addresses.
+    /// </summary>
+    internal class EmulatorData
+    {
+        /// <summary>
+        /// Serial of the running game.
+        /// For all TR1 version serials see: http://redump.org/discs/system/psx/letter/t/
+        /// Used to differentiate game versions.
+        /// </summary>
+        public StringWatcher serial { get; }
```

Comment:
Nitpick: if following [MS guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers), this should be Pascal case since it's public.
```suggestion
        public StringWatcher Serial { get; }
```

Reply:
Done.